apps/showcase/scripts/prepare-training-exercises/index.cjs

```diff
@@ -0,0 +1,26 @@
+const {readdir, readFile, writeFile} = require('node:fs/promises');
+const {getFilesTree} = require('@o3r/training-tools');
+const {dirname, join, resolve} = require('node:path');
+const glob = require('globby');
+
+/**
+ * The purpose of this script is to generate the exercise and solution files for the code editor
+ */
+
+void (async () => {
+  const root = resolve(__dirname, '..', '..');
+  const files = await glob('src/assets/trainings/*-training/**/(exercise|solution)/**', {cwd: root});
```

Comment:
If we are in a `trainings` folder 
we don't need to have a subfolder named `topic-training`, `topic` could be enough no ?
```suggestion
  const files = await glob('src/assets/trainings/**/(exercise|solution)/**', {cwd: root});
```